bundles/org.openhab.binding.bluetooth.radoneye/src/main/resources/OH-INF/update/instructions.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes" ?>
+<update:update-descriptions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:update="https://openhab.org/schemas/update-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/update-description/v1.0.0 https://openhab.org/schemas/update-description-1.0.0.xsd">
+	<thing-type uid="bluetooth:radoneye_rd200">
+		<instruction-set targetVersion="1">
+			<add-channel id="decay">
+				<type>radoneye_decay</type>
+				<label>Decay Counter</label>
+			</add-channel>
```

Comment:
You need also to update the channel where dimension was changed. Something like:
```suggestion
			</add-channel>
			<update-channel id="radon">
				<type>bluetooth:radoneye_radon</type>
			</update-channel>
```